packages/terra-date-picker/src/react-datepicker/month.jsx

```diff
@@ -124,7 +124,7 @@ export default class Month extends React.Component {
 
   render () {
     return (
-      <div tabIndex="0" className={this.getClassNames()} onMouseLeave={this.handleMouseLeave} role="listbox">
+      <div tabIndex="0" aria-label="Month" className={this.getClassNames()} onMouseLeave={this.handleMouseLeave} role="listbox">
```

Comment:
Need translations for this one as well.

Reply:
we can re-use terra-clinical-onset-picker's "month" translation: https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-onset-picker/translations/en.json#L11